DiscordBot/Data/IInitializeGuilds.cs

```diff
@@ -6,7 +6,7 @@ namespace DiscordBot.Data
 {
     public interface IInitializeGuilds
     {
-        Task AddNewGuild(SocketGuild guild);
+        Task<bool> AddNewGuild(SocketGuild guild);
```

Comment:
Just adds a guild to the server and returns true or false depending on whether or not it was successful.  I added the bool because I only want to add users if the guild was successfully added.